### DISCUSSION TITLE
Not found (not-found.tsx) not using root layout?

### DISCUSSION THREAD

**Discussion by michaelclaesson** [vercel/next.js#53352] [Help] [answered]
### Summary
Hi!

I'm new to Next.js and trying to get to grips with how it's working. Not doing anything advanced at the moment just getting some data from Strapi and displaying it. I have a few links that should result in a 404 and they do, so I've created a not-found.tsx in the app folder. For some reason it doesn't render my header, nor does it use the html lang attribute or the body class. Any idea why that is?

I'm expecting the output of not-found.tsx to be a child of the root layout but this doesn't seem to be the case.

### Additional information

**layout.tsx**
```
import './globals.scss'

import type { Metadata } from 'next'
import { Roboto } from 'next/font/google'
import Header from '@/src/partials/header'

const roboto = Roboto({ weight: ['400', '700'], subsets: ['latin'] })

export const metadata: Metadata = {
  title: 'Title',
  description: 'Description',
}

export default function RootLayout({
  children,
}: {
  children: React.ReactNode
}) {
  return (
    <html lang="en">
      <body className={roboto.className}>
        <div className="container mx-auto p-4 py-16">
          <Header />
          {children}
        </div>
      </body>
    </html>
  )
}
```

**not-found.tsx**
```
import type { Metadata } from 'next'

export const metadata: Metadata = {
  title: 'Test - Not found',
  description: 'Not found',
}

export default function NotFound() {
  return (
    <h1 className="text-5xl font-bold mb-4">Page not found</h1>
  )
}
```

**Comment by icyJoseph** [comment, marked as answer]
What next.js version do you have? I think in 13.4.10, and 13.4.11 this bug was present, but it was fixed in next@13.4.12

**Reply by michaelclaesson** [reply]
That was it! Thanks!

**Comment by roopakv** [comment]
I keep running into this issue with 13.4.19 as well. 

I am missing the head element which I have in my layout on all not found routes. Curious if anyone else has noticed this. 

Specifically I have a script in my layout which I mark as `beforeInteractive` and it loads much later in the case of `not-found`.

**Comment by Miloshinjo** [comment]
My fonts that are loaded in `rootLayout` are not being applied in the `not-found.tsx`, so I think this bug is still present.

Version of Next is `13.4.19`

**Comment by salikn** [comment]
I have the same issue, the `head` of the layout is not loading at 404 pages, only react <script> tags with async would be picked up at the 404 page
